src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray.cs

```diff
@@ -550,7 +548,7 @@ internal static ImmutableArray<T> CreateDefensiveCopy<T>(T[] items)
 
             // defensive copy
             var tmp = new T[items.Length];
-            Array.Copy(items, tmp, items.Length);
+            Array.Copy(items, 0, tmp, 0, items.Length);
```

Comment:
Yes, but after looking up the lower bounds of rank 0 (the overload can't assume the array starts at 0).